bikeshed/boilerplate/webappsec/status-CRD.include

```diff
@@ -0,0 +1,56 @@
+<p>
+  <em>This section describes the status of this document at the time of
+  its publication. Other documents may supersede this document. A list of
+  current W3C publications and the latest revision of this technical report
+  can be found in the <a href="https://www.w3.org/TR/">W3C technical reports
+  index at https://www.w3.org/TR/.</a></em>
+</p>
+
+<p>
+  This document was published by the
+  <a href="https://www.w3.org/2011/webappsec/">Web Application Security Working Group</a>
+  as a Candidate Recommendation Draft. This document is intended to become a W3C Recommendation.
+</p>
+
+<p>
+  The (<a href="https://lists.w3.org/Archives/Public/public-webappsec/">archived</a>) public mailing list
+  <a href="mailto:public-webappsec@w3.org?Subject=%5B[SHORTNAME]%5D%20PUT%20SUBJECT%20HERE">public-webappsec@w3.org</a>
+  (see <a href="https://www.w3.org/Mail/Request">instructions</a>)
+  is preferred for discussion of this specification.
+  When sending e-mail,
+  please put the text “[SHORTNAME]” in the subject,
+  preferably like this:
+  “[<!---->[SHORTNAME]] <em>…summary of comment…</em>”
+</p>
+
+<p>
+  Publication as a Candidate Recommendation does not imply endorsement by the
+  W3C Membership. A Candidate Recommendation Draft integrates changes from the
+  previous Candidate Recommendation that the Working Group intends to include in
+  a subsequent Candidate Recommendation Snapshot. This is a draft document and
+  may be updated, replaced or obsoleted by other documents at any time. It is
+  inappropriate to cite this document as other than work in progress.
+</p>
+
+<p>
+  The entrance criteria for this document to enter the Proposed Recommendation stage
+  is to have a minimum of two independent and interoperable user agents that
+  implementation all the features of this specification, which will be determined by
+  passing the user agent tests defined in the test suite developed by the Working
+  Group. The Working Group will prepare an implementation report to track progress.
+</p>
+
+<p>
+    This document was produced by a group operating under
+    the <a href="https://www.w3.org/Consortium/Patent-Policy-20200915/">W3C Patent Policy</a>.
+    W3C maintains a <a href="https://www.w3.org/groups/wg/das/ipr" rel=disclosure>public list of any patent disclosures</a>
```

Comment:
This should be https://www.w3.org/groups/wg/webappsec/ipr

Reply:
> This should be [w3.org/groups/wg/webappsec/ipr](https://www.w3.org/groups/wg/webappsec/ipr)

oops, thanks for catching that — will open another PR with a fix